src/components/sidebar/sidebar.module.css

```
@@ -48,6 +48,22 @@ a {
   margin-top: 5px;
 }
 
+.directoryNotice {
+  margin-top: 2px;
+  font-size: 11px;
+  color: var(--text);
+  word-wrap: break-word;
```

Stylelint (17.14.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)